plutus-benchmark/script-contexts/test/checkScriptContexEqualityData-20.budget.golden

```diff
@@ -1,2 +1,2 @@
-({cpu: 60538597
-| mem: 200402})
\ No newline at end of file
+({cpu: 65159541
```

Comment:
oof

Reply:
This is V3 against V1, not against V2, so slightly less bad!